nix/services/memcached_test.nix

```
@@ -0,0 +1,23 @@
+{ pkgs, config, ... }:
+{
+  services.memcached."memcached1".enable = true;
+
+  settings.processes.test =
+    let
+      cfg = config.services.memcached."memcached1";
+    in
+    {
+      command = pkgs.writeShellApplication {
+        runtimeInputs = [
+          cfg.package
+          pkgs.gnugrep
+          pkgs.netcat
+        ];
+        text = ''
+          echo -e "stats\nquit" | nc 127.0.0.1 11211 | grep -q "STAT version"
```

```suggestion
          echo -e "stats\nquit" | nc 127.0.0.1 11211 | grep "STAT version"
```
For more visibility in the logs:
```sh
memcached-test> + process-compose --no-server -t=false
memcached-test> [test   ] STAT version 1.6.27
memcached-test> + set +x
```